Dockerfile

```diff
@@ -93,3 +96,38 @@ RUN apk add --no-cache --virtual .build-deps \
                 && ls \
                 && make \
                 && make install
+
+## Adding Citus
+
+# Install Citus dependencies
+RUN apk add --no-cache --virtual .citus-deps \
+    curl \
+    jq
+
+# Install Citus
+ARG CITUS_VERSION="11.2.0"
```

Comment:
Yes, this can be a future task, I believe, as of now, we are in the development phase of adding extensions and if we add version variables to CI now itself, it would add more jobs, which I believe in the development phase is unnecessary.  Sticking to the smoke tests solves the purpose in the dev phase.